### DISCUSSION TITLE
Gas tracker... Abit screwy.

### DISCUSSION THREAD

**Discussion by jonathh21** [BottlecapDave/HomeAssistant-OctopusEnergy#543] [Q&A]
So it might be a coincidence,  but I spotted and started playing with enabling 'events' around gas and electricity and the next rates.

Now for the last 3 days my 'next_rate' has been unknown and the 'current_rate' has a long valid from/to.

I am very new to I gas tracker, and the price has been the same for those days... So it might be a quirk but I am trying to build integration's that make decisions based if the rate is cheaper tomorrow than today.

So I am not sure if I have broken something or if it is a glitch in the matrix.

**Comment by BottlecapDave** [comment]
Hello. As per the [docs](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/entities/gas.md#next-rate), this is by design. Next rate will only be populated if a future rate is different from the current rate
